README.markdown

```diff
@@ -0,0 +1,23 @@
+# OmniAuth Fishbrain
+
+OmniAuth strategy for Fishbrain
+
+## Installation
+
+```ruby
+gem 'omniauth-google-oauth2'
```

Comment:
This must be a copy+paste error

Reply:
lol, that was the initial reason I changed the readme, then forgot about the main point. 🐰